### PR TITLE
fix: offix server implementation

### PR DESCRIPTION
**Pull request by wtrocki**
Follow README to start.

Explore queries. For example

```
query test {
  findAllTasks{
   id
  }
}
```

### CODE REVIEW

**Comment by harshithdwivedi** [comment]
I'm assuming this has been tested on your end @wtrocki; so I'll approve this.

**Review by harshithdwivedi — APPROVED** [review]

**Comment by wtrocki** [comment]
Actually. I would love to get some input from you guys. Let me know if you like new API and if anything can be changed or extended.

File: sample/server/model/Tasks.graphql [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  version: Int
+  title: String!
+  description: String!
+  status: String!
+  creationMetadata: CreationMetadata
+  assignedTo: User
+}
+
+type CreationMetadata {
+   createdDate: String
```
Comment (harshithdwivedi) [inline sample/server/model/Tasks.graphql:12]: Maybe `createdAt` is a better choice of words here.

File: sample/server/model/Tasks.graphql [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+   createdBy: User
+ }
+
+type User {
+  id: ID!
+  firstName: String!
+  lastName: String!
+  title: String!
```
Comment (harshithdwivedi) [inline sample/server/model/Tasks.graphql:20]: Title for user?

**Review by harshithdwivedi — COMMENTED** [review]
minor nits

**Comment by harshithdwivedi** [comment]
mirror nits, so merging.

**Pull request merged**